sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_data_lake_service_client_async.pyi

```diff
@@ -0,0 +1,151 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: skip-file
+
+from datetime import datetime
+from typing import (
+    Any,
+    Dict,
+    Optional,
+    Union,
+)
+from types import TracebackType
+from typing_extensions import Self
+
+from azure.core import MatchConditions
+from azure.core.async_paging import AsyncItemPaged
+from azure.core.credentials import AzureNamedKeyCredential, AzureSasCredential
+from azure.core.credentials_async import AsyncTokenCredential
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.tracing.decorator_async import distributed_trace_async
+from .._models import (
+    AnalyticsLogging,
+    CorsRule,
+    DirectoryProperties,
+    EncryptionScopeOptions,
+    FileProperties,
+    FileSystemProperties,
+    Metrics,
+    PublicAccess,
+    RetentionPolicy,
+    StaticWebsite,
+    UserDelegationKey,
+)
+from .._shared.base_client import StorageAccountHostsMixin
+from .._shared.base_client_async import AsyncStorageAccountHostsMixin
+from ._data_lake_directory_client_async import DataLakeDirectoryClient
+from ._data_lake_file_client_async import DataLakeFileClient
+from ._data_lake_lease_async import DataLakeLeaseClient
+from ._file_system_client_async import FileSystemClient
+
+class DataLakeServiceClient(AsyncStorageAccountHostsMixin, StorageAccountHostsMixin):  # type: ignore [misc]
```

Comment:
Same nit as other PR-- just want to double check to see if we can do anything to improve behavior here.